tests/numeric/test_TwoColumnOperatorTransformer.py

```diff
@@ -19,6 +19,27 @@ def example_transformer():
 class TestTwoColumnOperatorTransformerInit:
     """Tests for TwoColumnMethodTransformer.__init__()."""
 
+    def test_column_type_error(self):
+        """Checks that an error is raised if the column type is not a list."""
+        with pytest.raises(
+            TypeError,
+            match="columns must be a list containing two column names",
+        ):
+            TwoColumnOperatorTransformer(
+                "mul",
+                "a, b",
+                "c",
+                pd_method_kwargs={"axis": 1},
+            )
+
+    def test_column_size_error(self):
+        """Checks that the column is of length 2."""
+        with pytest.raises(
+            ValueError,
+            match="columns must be a list containing two column names",
+        ):
+            TwoColumnOperatorTransformer("mul", [], "c", pd_method_kwargs={"axis": 1})
```

Comment:
Please can you check this with a list of length one? 

An empty list will evaluate to none and so could accidentally be picking up some other behaviour (I'm maybe being over cautious here)

Reply:
all good with both empty and length one it seems

Reply:
included both tests for empty and length 1 https://github.com/lvgig/tubular/pull/183/commits/a25f43968a110293324dbaa518dec1290a6b2117